<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cognition</class>
 <widget class="QMainWindow" name="Cognition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cognition</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openCamera">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenCamera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopCamera">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>CloseCamera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Catch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Catch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Histogram">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threshold">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>250</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Theshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>hold</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hold">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edgeHoldlow">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edgeHoldhigh">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remap">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>remap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Eroded">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Eroded</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dialated">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Dilated</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ErodeSize">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="DilateSize">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Location">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Recolor">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Recolor</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Original</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eroded</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dilated</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cErode">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Original</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>recolored</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cDilate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Original</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>recolored</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Cblur">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Original</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recolored</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cthre">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Original</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eroded</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dilated</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ChessPosition">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ChessPosition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chessColor">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ChessColor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChessMark">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ChessMark</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="positionHold">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ColorHold">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorCount">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>count[</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
